core/src/main/scala/kafka/coordinator/GroupMetadata.scala

```diff
@@ -128,6 +129,15 @@ case class GroupSummary(state: String,
                         members: List[MemberSummary])
 
 /**
+ * Case class used to represent group metadata (including the group coordinator) for the DescribeGroup API
+ */
+case class ConsumerGroupSummary(state: String,
+                                protocolType: String,
```

Comment:
Yes, it'll be `consumer`, but there is a check [here](https://github.com/vahidhashemian/kafka/blob/904bcee920605e1fe5417fe39b5397194b665832/core/src/main/scala/kafka/admin/AdminClient.scala#L151) to verify that the protocol type os valid. If we remove this, I think that check has to be removed too. Should I still go ahead and remove it?